agent/installer/installer.go

```diff
@@ -52,14 +52,18 @@ func getSupportedRegistry(ob algo.OutputBuilder) registry {
 
 		// BYOH Bundle Repository. Associate bundle with installer
 		linuxDistro := "Ubuntu_20.04.1_x86-64"
+		addBundleInstaller(linuxDistro, "v1.21.*", &algo.Ubuntu20_4K8s1_22{})
```

Comment:
Question: Do we need to use a specific minor k8s version in the name. Or we could use a type alias
```
type Ubuntu2004_k8s algo.Ubuntu20_4K8s1_22
```

Reply:
We can do that, sounds good and reasonable. @anusha94, @georgievaVMW  what is your opinion?

Reply:
> Do we need to use a specific minor k8s version in the name

@dharmjit @jeuxdemains 
Good suggestion and I think this will work for now. The reason for including minor in the name was what if certain steps / sequence of steps change for a future K8s version? Then we would want to define another algo that has the new set of steps. But this could be rare, so we will end up using the same `Ubuntu20_4K8s1_22` algo as long as the steps to install k8s remain the same.
I'd leave it to you folks to decide, don't have a strong opinion :)